build-system/compile/compile.js

```diff
@@ -13,7 +13,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-'use strict';
```

Comment:
Was this intentionally removed? It's in all our files.

Reply:
Addressed in b66b19e.